helix-cli/src/commands/integrations/helix.rs

```diff
@@ -153,7 +154,7 @@ impl<'a> HelixManager<'a> {
         Ok(())
     }
 
-    pub(crate) async fn deploy(&self, path: Option<String>, cluster_name: String) -> Result<()> {
+    pub(crate) async fn deploy(&self, path: Option<String>, cluster_name: String, dev_profile: Option<bool>) -> Result<()> {
```

Comment:
should probably pass in the BuildMode here to not special case the API for Dev in case we add more profiles